src/Exceptions/HttpVersionNotSupported.php

```diff
@@ -2,10 +2,10 @@
 
 namespace Tnapf\Router\Exceptions;
 
-class HttpHTTPVersionNotSupported extends HttpException {
+class HttpVersionNotSupported extends HttpException
+{
     public const CODE = 505;
-    public const PHRASE = "HTTP Version Not Supported";
+    public const PHRASE = "Version Not Supported";
```

Comment:
Due to how the generation works it uses the phrase for the class name so probably what I have to do is add some code in the generation script that removes http from the beginning of the phrase when naming the class rather than me removing it from the json of httpcodes

Reply:
Just throw out the generator, you have the classes you need, will there really be any more that get added? If yes, is it really going to be *that* many more that its worth to keep the generator?

Reply:
I plan on moving the httpcodes and classes to their own repository. Which will also have all codes (not just 400-600) turned into classes. So eventually this generator will be removed from this repo and most likely refactored a bit more in its final home.